client/analytics/report/categories/index.js

```diff
@@ -0,0 +1,44 @@
+/** @format */
+/**
+ * External dependencies
+ */
+import { Component, Fragment } from '@wordpress/element';
+import PropTypes from 'prop-types';
+
+/**
+ * Internal dependencies
+ */
+import { charts, filters } from './config';
+import getSelectedChart from 'lib/get-selected-chart';
+import { ReportFilters } from '@woocommerce/components';
```

Comment:
I think we are separating dependencies from `@woocommerce` packages in a different block called `WooCommerce dependencies`. Like this:
https://github.com/woocommerce/wc-admin/blob/master/client/header/index.js#L13